server/routes.json

```diff
@@ -43,5 +43,10 @@
         "pattern": "/wedo",
         "view": "wedo2",
         "title": "LEGO WeDo 2.0"
+    },
+    {
+        "pattern": "/guidelines",
```

Comment:
The url currently used is `/community_guidelines`, so we should continue with that at least for now, just while we're migrating. Otherwise, existing links in the django repo will break once this is in production.